code/model/DMSDocumentSet.php

```diff
@@ -49,8 +62,10 @@ public function getCMSFields()
     {
         // PHP 5.3 only
         $self = $this;
-
         $this->beforeUpdateCMSFields(function (FieldList $fields) use ($self) {
+            $fields->removeFieldsFromTab('Root.Main',
```

Comment:
Fixed (as in dropped to new line). Btw running phpcbf --standard=PSR2 doesn't drop this to a new line automatically.